example/themes/original/src/components/GiftCard/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
```

Comment:
can you create only one folder for the Gift cards and inside you can put all the components you need? Try to optimize your code with best practice